### PR TITLE
fix(banner): revise title style and banner component

### PR DESCRIPTION
**Pull request by SivanYeh**
## Types of changes

* **New feature**

## Steps to Test This Pull Request

1. Adjust screen width
2. Scroll down
3. Zh/En

## Expected behavior

image
## Related Issue

#230

### CODE REVIEW

**Comment by flynnhou** [comment]
@SivanYeh Please provide PR description otherwise it is too painful to review.

**Comment by flynnhou** [comment]
When language is EN, it looks off:

| PR | Figma |
|----|-------|
| image | image |

File: components/core/layout/Banner.vue [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
-                items-start
-                mx-8
-                w-full
-                text-xs
-                md:w-3/6 md:text-base
-            "
-        >
-            <slot name="text"></slot>
+        <div class="mainstyle vertstyle">
+            <slot name="standard"></slot>
```
Comment (flynnhou) [inline components/core/layout/Banner.vue:7]: Can you explain what's the rationale behind this change?
I'll be providing a better solution to optimize tailwind's rwd soon. In the meantime, see if you can find some clues from this merged PR: https://github.com/pycontw/pycontw-frontend/pull/220.

**Review by flynnhou — CHANGES_REQUESTED** [review]
Besides the code, the images aren't as ideal.

1. I don't think we need another `BannerBody.svg` because it seems like the entire app's background color, no?
2. Need to rename `RWD2.png` and `WEB2.png`:
   - Why all uppercase? Rename it to kebab case.
   - RWD2 and WEB2 don't reflect the meaning behind the images. They should be named as something like `banner-mobile.png` and `banner-desktop.png`, respectively.
   - Consider to move them under `static/img/background/`
3. These are background images and are displayed as assets. Their current sizes are too large for their lower significance. See if you can export the images to make them at least less than 1MB, the smaller the better. Image sizes will affect the performance and user experience of the website. If you don't know how to make this happen, leave it as it is for now.

**Comment by mattwang44** [comment]
close this PR as #266 is merged

**Pull request closed without merging**